TYPO3.Neos.NodeTypes/Resources/Private/TypoScript/Root.ts2

```diff
@@ -63,9 +63,11 @@ prototype(TYPO3.Neos.NodeTypes:MultiColumn) < prototype(TYPO3.Neos:Content) {
 	layout = ${q(node).property('layout')}
 	attributes.class = ${'container columns-' + q(node).property('layout')}
 	columns = TYPO3.TypoScript:Collection {
+		@context.columnLayout = ${q(node).property('layout')}
```

Comment:
just out of curiosity:.
i followed your slack discussion with @dimaip about best practices in TS
According to this:
https://neos-project.slack.com/archives/neos-general/p1464708059007109 it would be better to add layout to context. Or am i confounding things here?